packages/expo-file-system/src/next/FileSystem.types.ts

```diff
@@ -0,0 +1,21 @@
+export type Path = string;
+
+export declare class Directory {
+  constructor(path: Path);
+  path: string;
+  validatePath(): void;
+  delete();
+  exists(): boolean;
+  create();
+}
+
+export declare class File {
+  constructor(path: Path);
+  path: string;
+  validatePath();
+  text(): string;
+  write(content: string): any;
+  delete();
+  exists(): boolean;
+  create();
+}
```

Comment:
Missing doc comments and return types (I mean void functions, only `validatePath` has it declared)